quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/QuarkusProfileConfigResolver.java

```diff
@@ -1,17 +1,73 @@
 package org.keycloak.quarkus.runtime;
 
+import org.keycloak.common.Profile;
 import org.keycloak.common.profile.CommaSeparatedListProfileConfigResolver;
+import org.keycloak.common.profile.ProfileConfigResolver;
+import org.keycloak.common.profile.SingleProfileConfigResolver;
+import org.keycloak.config.FeatureOptions;
+import org.keycloak.config.WildcardOptionsUtil;
+import org.keycloak.quarkus.runtime.cli.PropertyException;
 import org.keycloak.quarkus.runtime.configuration.Configuration;
 
-public class QuarkusProfileConfigResolver extends CommaSeparatedListProfileConfigResolver {
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.keycloak.quarkus.runtime.configuration.MicroProfileConfigProvider.NS_KEYCLOAK_PREFIX;
+
+public class QuarkusProfileConfigResolver implements ProfileConfigResolver {
+    private final CommaSeparatedListProfileConfigResolver commaSeparatedResolver;
+    private final SingleProfileConfigResolver singleResolver;
```

Comment:
It might be better to do this is to update the Environment - to add a Quarkus specific SingleProfileConfigResolver ahead of the QuarkusProfileConfigResolver. Especially if the longer-term plan is to refactor the PropertiesProfileConfigResolver so that it is SingleProfileConfigResolver.

Reply:
@shawkins Good point. Updated:  https://github.com/keycloak/keycloak/pull/43542/commits/2eb58560ce56aebe507324cf14f52f71e57277c1